app/src/plugins/vuetify.js

```diff
@@ -6,6 +6,12 @@ import { THEME_DARK_BLACK_NAME, THEME_LIGHT_BLUE_NAME } from '../utils/enums'
 
 Vue.use(Vuetify)
 
+import { languages } from './i18n'
+
+let userLanguage = window.navigator.userLanguage || window.navigator.language
+userLanguage = userLanguage.split('-')
+userLanguage = userLanguage[0]
+
```

Comment:
check if we support this language. if not, fallback to en

Reply:
> check if we support this language. if not, fallback to en

Done